src/styles/main.scss

```diff
@@ -0,0 +1,43 @@
+@import './utils/mixin';
+@import './utils/variables';
+
+.page {
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: ($cellSize * $weakSize) +6px;
```

Comment:
There seems to be a typo in the variable name `$weakSize`; it should be `$weekSize` if it represents the number of days in a week. Please correct the variable name to match its intended meaning.